llvm/test/tools/llvm-readobj/ELF/dynamic-tags.test

```diff
@@ -138,6 +140,335 @@
+# JSON64-NEXT:      "Value": 4096,
+# JSON64-NEXT:      "Type": "INIT"
+# JSON64-NEXT:  },
+# JSON64-NEXT:  {
+# JSON64-NEXT:      "Tag": 13,
+# JSON64-NEXT:      "Value": 4096,
+# JSON64-NEXT:      "Type": "FINI"
+# JSON64-NEXT:  },
+# JSON64-NEXT:  {
+# JSON64-NEXT:      "Tag": 14,
+# JSON64-NEXT:      "Value": 3,
+# JSON64-NEXT:      "Name": "U",
+# JSON64-NEXT:      "Type": "SONAME"
+# JSON64-NEXT:  },
+# JSON64-NEXT:  {
+# JSON64-NEXT:      "Tag": 15,
+# JSON64-NEXT:      "Value": 5,
+# JSON64-NEXT:      "Path": [
+# JSON64-NEXT:          "f"
+# JSON64-NEXT:      ],
+# JSON64-NEXT:      "Type": "RPATH"
+# JSON64-NEXT:  },
+# JSON64-NEXT:  {
+# JSON64-NEXT:      "Tag": 16,
+# JSON64-NEXT:      "Value": 1311768467294899695,
+# JSON64-NEXT:      "Type": "SYMBOLIC"
+# JSON64-NEXT:  },
+# JSON64-NEXT:  {
+# JSON64-NEXT:      "Tag": 17,
+# JSON64-NEXT:      "Value": 4096,
+# JSON64-NEXT:      "Type": "REL"
+# JSON64-NEXT:  },
+# JSON64-NEXT:  {
+# JSON64-NEXT:      "Tag": 18,
+# JSON64-NEXT:      "Value": 16,
+# JSON64-NEXT:      "Type": "RELSZ"
+# JSON64-NEXT:  },
+# JSON64-NEXT:  {
+# JSON64-NEXT:      "Tag": 19,
+# JSON64-NEXT:      "Value": 291,
+# JSON64-NEXT:      "Type": "RELENT"
+# JSON64-NEXT:  },
+# JSON64-NEXT:  {
+# JSON64-NEXT:      "Tag": 20,
+# JSON64-NEXT:      "Value": 7,
+# JSON64-NEXT:      "Type": "PLTREL"
+# JSON64-NEXT:  },
+# JSON64-NEXT:  {
+# JSON64-NEXT:      "Tag": 21,
+# JSON64-NEXT:      "Value": 18364757930599072545,
+# JSON64-NEXT:      "Type": "DEBUG"
+# JSON64-NEXT:  },
+# JSON64-NEXT:  {
+# JSON64-NEXT:      "Tag": 22,
+# JSON64-NEXT:      "Value": 1234605616436508552,
+# JSON64-NEXT:      "Type": "TEXTREL"
+# JSON64-NEXT:  },
+# JSON64-NEXT:  {
+# JSON64-NEXT:      "Tag": 23,
+# JSON64-NEXT:      "Value": 4096,
+# JSON64-NEXT:      "Type": "JMPREL"
+# JSON64-NEXT:  },
+# JSON64-NEXT:  {
+# JSON64-NEXT:      "Tag": 24,
+# JSON64-NEXT:      "Value": 9833440827789222417,
+# JSON64-NEXT:      "Type": "BIND_NOW"
+# JSON64-NEXT:  },
+# JSON64-NEXT:  {
+# JSON64-NEXT:      "Tag": 25,
+# JSON64-NEXT:      "Value": 4096,
+# JSON64-NEXT:      "Type": "INIT_ARRAY"
+# JSON64-NEXT:  },
+# JSON64-NEXT:  {
+# JSON64-NEXT:      "Tag": 26,
+# JSON64-NEXT:      "Value": 4096,
+# JSON64-NEXT:      "Type": "FINI_ARRAY"
+# JSON64-NEXT:  },
+# JSON64-NEXT:  {
+# JSON64-NEXT:      "Tag": 27,
+# JSON64-NEXT:      "Value": 16,
+# JSON64-NEXT:      "Type": "INIT_ARRAYSZ"
+# JSON64-NEXT:  },
+# JSON64-NEXT:  {
+# JSON64-NEXT:      "Tag": 28,
+# JSON64-NEXT:      "Value": 16,
+# JSON64-NEXT:      "Type": "FINI_ARRAYSZ"
+# JSON64-NEXT:  },
+# JSON64-NEXT:  {
+# JSON64-NEXT:      "Tag": 29,
+# JSON64-NEXT:      "Value": 7,
+# JSON64-NEXT:      "Path": [
+# JSON64-NEXT:          "w"
+# JSON64-NEXT:      ],
+# JSON64-NEXT:      "Type": "RUNPATH"
+# JSON64-NEXT:  },
+# JSON64-NEXT:  {
+# JSON64-NEXT:      "Tag": 30,
+# JSON64-NEXT:      "Value": 18446744073709551615,
+# JSON64-NEXT:      "Flags": "ORIGIN SYMBOLIC TEXTREL BIND_NOW STATIC_TLS ",
```

Comment:
trailing space may look strange

Reply:
Yeah. It seems like a bug in the dynamic string method. I'll find it and squash it. It doesn't show up in the LLVM formatter since there aren't quotes to delineate it

Reply:
I would have thought that Flags should be a list anyway?

Reply:
yeah. done